presto-main/src/test/java/com/facebook/presto/sql/planner/TestDynamicFilter.java

```diff
@@ -73,6 +73,19 @@ public void testEmptyJoinCriteria()
                                 exchange(tableScan("lineitem")))));
     }
```

Comment:
Please add more tests to make sure the new logic in `PredicatePushDown.java` works as intended, e.g.:
https://github.com/trinodb/trino/pull/2674/commits/7aa4b22b40f8665e104f7b53dd64e19e831d0f86#diff-a4f192ad8b8e01837c8acc620aeef7beae742ef520df17f26745466d3930067a
https://github.com/trinodb/trino/pull/2674/commits/7aa4b22b40f8665e104f7b53dd64e19e831d0f86#diff-8b52634ab295ddb09b2bb605b7772b03f9744ed6417c2b02a3308917502a6d80